exa.py

```diff
@@ -294,6 +312,5 @@ def parse(self, statements):
     with open(args.file, 'r') as f:
         statements = f.readlines()
 
-    interp = Interpreter()
-    result = interp.run(statements)
+    result = Interpreter().run(statements)
```

Comment:
I'm not sure why you changed these lines?

Reply:
I'll make the changes above,  as for the clean up I did that earlier and forgot about it, will do. As for interp, this instance object isn't used anywhere else to access any other methods of Interpreter class so I just removed it. My apologies.

Reply:
No need to apologize, I was just curious. It's a style difference and I'm not sure I can explain why I like having the interpreter reference in its own variable.

Note I've pushed some changes this afternoon that mean you'll need to rebase this and update the init() method of the new class.